save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/inputform/InputForms.kt

```diff
@@ -10,6 +10,7 @@ import com.saveourtool.save.validation.*
 
 private const val URL_PLACEHOLDER = "https://example.com"
 private const val EMAIL_PLACEHOLDER = "test@example.com"
+private const val SEVERITY_VECTOR_PLACEHOLDER = "CVSS:3.1/AV:_/AC:_/PR:_/UI:_/S:_/C:_/I:_/A:_"
```

Comment:
So no tooltip/popover/other explanation is not present, is it?

Reply:
Added